source/connect/disconnect.txt

```diff
@@ -6,8 +6,14 @@ Disconnect Connection
 
 .. default-domain:: mongodb
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+Disconnecting from a MongoDB deployment closes the Compass connection to
```

Comment:
Hey @jocelyn-mendez1 , sorry for the late thought here, but for parallelism with the "Connect to MongoDB" page title, maybe we should retitle this page to "Disconnect from MongoDB".